ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -192,6 +192,10 @@
            (variables/delete-variable
             provider-id native-id request)))
        ;; Services
+       (context ["/validate/service/:native-id" :native-id #".*$"] [native-id]
```

Comment:
I think we just have to add documentation in the ingest docs

Reply:
Yes. I'm working on it now. Thanks.